jobs/build/ocp4/build.groovy

```diff
@@ -347,6 +348,12 @@ def stageBuildImages() {
         def r = buildlib.determine_build_failure_ratio(recordLog)
         if (r.total > 10 && r.ratio > 0.25 || r.total > 1 && r.failed == r.total) {
             echo "${r.failed} of ${r.total} image builds failed; probably not the owners' fault, will not spam"
+            if (r.failed == r.total) {
+                allImagebuildfailed = true
+                commonlib.slacklib.to(params.BUILD_VERSION).say("""
+                    *:warning:(test-run) all of ${r.total} image builds failed in ocp4 job*
```

Comment:
obviously, remove `test-run` before merging :)
also could this link to the job?